docs/book/v6/tutorials/create-book-module.md

```diff
@@ -523,197 +464,284 @@ declare(strict_types=1);
-use Mezzio\Hal\Metadata\MetadataMap;
-use Api\App\ConfigProvider as AppConfigProvider;
 
 class ConfigProvider
 {
     public function __invoke(): array
     {
         return [
             'dependencies'     => $this->getDependencies(),
-            'doctrine'     => $this->getDoctrineConfig(),
-            MetadataMap::class => $this->getHalConfig(),
+            'doctrine'         => $this->getDoctrineConfig(),
         ];
     }
 
     private function getDependencies(): array
     {
         return [
-            'delegators' => [
-                Application::class => [
-                    RoutesDelegator::class
-                ]
-            ],
             'factories' => [
-                BookHandler::class    => AttributedServiceFactory::class,
-                BookService::class    => AttributedServiceFactory::class,
                 BookRepository::class => AttributedRepositoryFactory::class,
             ],
-            'aliases'   => [
-                BookServiceInterface::class     => BookService::class,
-            ],
         ];
     }
 
     private function getDoctrineConfig(): array
     {
         return [
             'driver' => [
-                'orm_default'   => [
+                'orm_default'  => [
                     'drivers' => [
-                        'Api\Book\Entity' => 'BookEntities'
+                        'Core\Book\Entity' => 'BookEntities',
                     ],
                 ],
-                'BookEntities'  => [
+                'BookEntities' => [
                     'class' => AttributeDriver::class,
                     'cache' => 'array',
-                    'paths' => __DIR__ . '/Entity',
+                    'paths' => [__DIR__ . '/Entity'],
                 ],
             ],
         ];
     }
+}
+```
 
-    private function getHalConfig(): array
-    {
-        return [
-            AppConfigProvider::getCollection(BookCollection::class, 'books.list', 'books'),
-            AppConfigProvider::getResource(Book::class, 'book.show')
-        ];
-    }
+### Registering the module
 
-}
+* register the module config by adding `Api\Book\ConfigProvider::class` and `Core\Book\ConfigProvider::class` in `config/config.php` under the `Api\User\ConfigProvider::class`
+* register the namespace by adding this line `"Api\\Book\\": "src/Book/src/"` and `"Core\\Book\\": "src/Core/src/Book/src/"`, in composer.json under the autoload.psr-4 key
+* update Composer autoloader by running the command:
+
+```shell
+composer dump-autoload
 ```
 
+That's it. The module is now registered.
+
+We need to configure access to the newly created endpoints, add `books::list-books`, `book::view-book` and `book::create-book` to the authorization rbac array, under the `UserRole::ROLE_GUEST` key.
+> Make sure you read and understand the rbac [documentation](https://docs.dotkernel.org/dot-rbac-guard/v4/configuration/).
+
 ## Migrations
 
 We created the `Book` entity, but we didn't create the associated table for it.
```

Comment:
In this section:

- fix typo `shel`
- replace `php bin/doctrine orm:validate-schema` with `php ./bin/doctrine orm:validate-schema`